### PR TITLE
Tablets optimizations

### LINKED ISSUE #722: Bigger folder and file icons
In the file list when in grid mode display the folder and files icons bigger to match the prototype

### LINKED ISSUE #723: Make thumbnails fill whole card
Make thumbnails fill whole card in the file list when in grid mode. Currently stops filling the whole card after going out of the screen?

### LINKED ISSUE #724: Cap bottom navigation bar's width
When the screen is big enough, the nav bar should not try to extend to the edge of the screen

### LINKED ISSUE #726: Empty placeholders should not be cut by the edge of the screen
Screenshot_20220331-151730_kDrive

### LINKED ISSUE #727: Card size in home's offline files should have the correct size according to prototypes

### LINKED ISSUE #728: Remove the size limit on drive selection button in home screen

### PR DESCRIPTION
**Pull request by LunarX**
Closes #728
Closes #727
Closes #726
Closes #724
Closes #723
Closes #722

### CODE REVIEW

File: app/src/main/java/com/infomaniak/drive/ui/fileList/FileAdapter.kt [hunk 1]
```diff
@@ -262,7 +263,10 @@ open class FileAdapter(
             val isGrid = viewHolderType == DisplayType.GRID
 
             if (isGrid) {
-                if (isHomeOffline) (layoutParams as ConstraintLayout.LayoutParams).dimensionRatio = "2:1"
+                if (isHomeOffline) {
+                    (layoutParams as ConstraintLayout.LayoutParams).dimensionRatio = null
+                    (layoutParams as ConstraintLayout.LayoutParams).height = 150.toPx()
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/ui/fileList/FileAdapter.kt:268]:
```suggestion
                    (layoutParams as ConstraintLayout.LayoutParams).apply {
                        dimensionRatio = null
                        height = 150.toPx()
                    }
```

File: app/src/main/java/com/infomaniak/drive/utils/FileItemUtils.kt [hunk 2]
```diff
@@ -118,16 +118,23 @@ private fun View.displayFileIcon(file: File, isGrid: Boolean) {
     val isGraphic = fileType == ConvertedType.IMAGE || fileType == ConvertedType.VIDEO
 
     when {
-        file.hasThumbnail && (isGrid || isGraphic) -> filePreview.loadAny(file.thumbnail(), fileType.icon)
+        file.hasThumbnail && (isGrid || isGraphic) -> {
+            filePreview.scaleType = ImageView.ScaleType.CENTER_CROP
+            filePreview.loadAny(file.thumbnail(), fileType.icon)
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/utils/FileItemUtils.kt:123]:
```suggestion
            filePreview.apply {
                scaleType = ImageView.ScaleType.CENTER_CROP
                loadAny(file.thumbnail(), fileType.icon)
            }
```

File: app/src/main/java/com/infomaniak/drive/utils/FileItemUtils.kt [hunk 3]
```diff
@@ -118,16 +118,23 @@ private fun View.displayFileIcon(file: File, isGrid: Boolean) {
...
                 val bitmap = context.getLocalThumbnail(file)
                 withContext(Dispatchers.Main) {
                     if (filePreview?.isVisible == true && context != null) filePreview.loadAny(bitmap, fileType.icon)
                 }
             }
         }
-        else -> filePreview.load(fileType.icon)
+        else -> {
+            filePreview.scaleType = ImageView.ScaleType.CENTER
+            filePreview.load(fileType.icon)
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/utils/FileItemUtils.kt:136]:
```suggestion
            filePreview.apply {
                scaleType = ImageView.ScaleType.CENTER
                load(fileType.icon)
            }
```

File: app/src/main/res/layout/fragment_home.xml [hunk 4]
```diff
@@ -62,6 +62,7 @@
                             android:gravity="start|center"
                             android:lineSpacingMultiplier="1"
                             android:maxLines="2"
+                            android:maxWidth="1000000dp"
```
Comment (KevinBoulongne) [inline app/src/main/res/layout/fragment_home.xml:65]: image

Comment (KevinBoulongne) [reply app/src/main/res/layout/fragment_home.xml:65]: image

Comment (KevinBoulongne) [inline app/src/main/res/layout/fragment_home.xml:65]: image

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_home.xml (hunk 4 above)
Comment (LunarX) [reply app/src/main/res/layout/fragment_home.xml:65]: jackie

**Review by JorisBodin — APPROVED** [review]

**Pull request merged**